cmd/tailscale/cli/web.go

```diff
@@ -94,6 +94,7 @@ func runWeb(ctx context.Context, args []string) error {
 		}
 		return nil
 	}
+	log.Printf("web server running on: %s\n", webArgs.listen)
```

Comment:
Assuming this compiles, try this?
```suggestion
	host, port, _ := net.SplitHostPort(webArgs.listen)
	if host == "" {
		host = "127.0.0.1"
	}
	log.Printf("web server running on: http://%s", net.JoinHostPort(host, port))
```

Reply:
or ideally pulled out into a `func urlOfListenAddr`